test/unit/controllers/SubmissionsControllerSpec.scala

```diff
@@ -105,6 +117,17 @@ class SubmissionsControllerSpec extends ControllerSpec {
         redirectLocation(result) must be(Some(controllers.routes.SubmissionsController.displayListOfSubmissions().url))
       }
 
+      def inTheLast(timeout: Timeout): BeMatcher[Instant] = new BeMatcher[Instant] {
```

Comment:
Should we not be in control of time so that we can avoid potentially assertions like this?  I though AppConfig had a clock defined, but apparently not.

Reply:
Good point, I'll make it unit testable with a fixed clock